CONTRIBUTING.md

```diff
@@ -1,47 +1,197 @@
+  - [Step 6: Pull Request](#step-5-pull-request)
+  - [Step 7: Merge](#step-6-merge)
+- [Troubleshooting](#troubleshooting)
+- [Related Repositories](#related-repositories)
+- [Finding Contributions](#finding-contributions-to-work-on)
+- [Code of Conduct](#code-of-conduct)
+- [Security Issue Notifications](#security-issue-notifications)
+- [Licensing](#licensing)
 
-Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
-information to effectively respond to your bug report or contribution.
+For a brief history and our ethos/design philosophy around the Amplify project please refer to this [document](/ETHOS.md).
 
+## Getting Started
 
-## Reporting Bugs/Feature Requests
+To get started with the Amplify Framework for iOS, first make sure you have the latest version of Xcode installed as well as cocoapods. You can install cocoapods by simply running:
 
-We welcome you to use the GitHub issue tracker to report bugs or suggest features.
+```
+sudo gem install cocoapods
+```
 
-When filing an issue, please check [existing open](https://github.com/awslabs/amplify-ios/issues), or [recently closed](https://github.com/awslabs/amplify-ios/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), issues to make sure somebody else hasn't already
-reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
+Then make sure you fork the project first and then clone it by running:
 
-* A reproducible test case or series of steps
-* The version of our code being used
-* Any modifications you've made relevant to the bug
-* Anything unusual about your environment or deployment
+```
+git clone git@github.com:YOURGITHUBUSERNAME/amplify-ios.git
+```
+GitHub provides additional documentation on [forking a repository](https://help.github.com/articles/fork-a-repo/).
 
+The Amplify iOS framework has been divided into multiple Xcode projects:
 
-## Contributing via Pull Requests
-Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
+- Core - Amplify.xcworkspace - Includes high level category interfaces and shared interfaces. Shared components including Hub.
+- API - AmplifyPlugins/API/APICategoryPlugin.xcworkspace - Includes the plugin with implementations for the API category. 
+- Analytics - AmplifyPlugins/Analytics/AnalyticsCategoryPlugin.xcworkspace - Includes the plugin with implementations for the analytics category.
+- DataStore - AmplifyPlugins/DataStore/DataStoreCategoryPlugin.xcworkspace - Includes the plugins with the implementations for the datastore category
+- Predictions - AmplifyPlugins/Predictions/PredictionsCategoryPlugin.xcworkspace - Includes 2 plugins (AWS and CoreML) with the implementations for the predictions category.
+- Storage - AmplifyPlugins/Storage/StoragePlugin.xcworkspace - Includes the plugin with the implementation for the storage category.
```

Comment:
I wonder if we need to reference the project files here? I'm afraid they might get out-of-sync if we refactor and move them around. I would say just the name of the categories should be enough so we don't create cross-dependencies of this guide and the filesystem.